projects/gomods/athens/upload-athens-artifacts.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+set -euo pipefail
+
+err_report() {
+    echo "Exited with error on line $1"
+}
+trap 'err_report $LINENO' ERR
+
+SCRIPT_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
+BUILD_DIR="$SCRIPT_ROOT/build"
+REPO=$1
+CHART_BUCKET=$2
+BASE_IMAGE=$3
+IMAGE=$4
+UPLOAD=$5
+
+mkdir -p $BUILD_DIR
+cd $BUILD_DIR
+
+git clone $REPO
+cd athens
+if [[ $UPLOAD == "true" ]]
+then
```

Comment:
Since this is nearly the same as the `else` condition, can you make an `OUTPUT` variable here and just run buildctl with 

```suggestion
OUTPUT="dest=/tmp/athens.tar"
if [[ $UPLOAD == "true" ]]; then
    OUTPUT="push=true"
fi
buildctl build \
   ... \
  --output type=oci,oci-mediatypes=true,name=$(IMAGE),$OUTPUT
```